mono/mini/aot-compiler.c

```diff
@@ -172,6 +172,8 @@ typedef struct MonoAotOptions {
 	gboolean no_direct_calls;
 	gboolean use_trampolines_page;
 	gboolean no_instances;
+	gboolean dedup; // We are collecting inflated methods and emitting non-inflated
+	char *dedup_include; // We are emitting inflated methods only
```

Comment:
Flesh out this comment, please:  I've read through the entire PR and I still don't know what the argument to `dedup-include=` is supposed to be.